src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Reliability/SqlConfigurableRetryLogicManager.cs

```diff
@@ -15,18 +16,27 @@ internal sealed class SqlConfigurableRetryLogicManager
     {
         private const string TypeName = nameof(SqlConfigurableRetryLogicManager);
 
+#if NET6_0_OR_GREATER
+        [UnconditionalSuppressMessage("Trimming", "IL2026:Members annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code",
+            Justification = "Usages of this property annotated with RequiresUnreferencedCode")]
```

Comment:
This feels wrong - if the usages are all annotated, then why not simply mark this with `RequiresUnreferencedCode`?

Also of note - this suppression will not work in the trimmer. The initialization code is actually part of the static constructor for the class in IL. This is a long standing problem we haven't been able to fix yet.

The simplest solution is probably to add an explicit static constructor, initialize the values there and suppress the warnings there as well.